libraries/WebServer/examples/WebServer/README.md

```diff
@@ -10,6 +10,12 @@ This example requires some space for a filesystem and runs fine boards with 4 MB
 * Partition Scheme: Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS)
     but LittleFS will be used in the partition (not SPIFFS)
 
+or
+
+* Board: Arduino Nano ESP32
+* Partition Scheme: "With FAT partition"
+
+
```

Comment:
This section is not needed at all, as you can run it on any board having a ESP32.
```suggestion
```

Can you please remove it and maybe change the section above to something like this:
```
This example requires some space for a filesystem and runs fine on supported SoCs with 4 MByte or more flash by selecting proper partition table:

* For using SPIFFS(LittleFS): Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS)
* For using FATFS: Default 4MB with ffat (1.2MB APP/1.5MB FATFS)

```